DMCompiler/Compiler/DM/DMParser.cs

```diff
@@ -1362,6 +1362,7 @@ public DMASTProcStatementLabel Label(DMASTIdentifier expression)
         public DMASTCallParameter[] CallParameters() {
             List<DMASTCallParameter> parameters = new();
             DMASTCallParameter parameter = CallParameter();
+            BracketWhitespace();
```

Comment:
This will only check for whitespace before the first comma. There needs to be another `BracketWhitespace()` call at the end of the while loop below for the following code to work

```
proc_call(1
    ,2
    ,3
    ,4)
```